### PR TITLE
Grouped visualization changes

### PR DESCRIPTION
**Pull request by nandeeshp24**
* Integrated grouping logic for bar chart and bubble chart with generic workflow
* Labels for scatterplot matrix data points made mandatory
* Reference column for coloring the data points on scatterplot matrix made optional
* Added display names and questions to get column values for viz parameters to RDF

### CODE REVIEW

File: backend-server/src/main/java/org/dice/ida/util/RDFUtil.java [hunk 1]
```diff
@@ -366,4 +370,92 @@ public String addDatasetName(String dsName) {
...
+			return paramDisplayMessageMap;
+		}
+		paramDisplayMessageMap = new HashMap<>();
+		String queryString = IDAConst.IDA_SPARQL_PREFIX +
+				"SELECT ?paramLabel ?displayMessage " +
+				"WHERE { " +
+				"    ?s rdf:type ivoc:Parameter ; " +
+				"       rdfs:label ?paramLabel ; " +
+				"       ivodp:hasDisplayMessage ?displayMessage " +
+				"}";
```
Comment (nikit91) [inline backend-server/src/main/java/org/dice/ida/util/RDFUtil.java:414]: Better to use StringBuilder here. @nandeeshp24 could you please do it for all the similar scenarios?

**Review by nikit91 — CHANGES_REQUESTED** [review]
please do String concatenation through StringBuilder.

File: backend-server/src/main/java/org/dice/ida/util/RDFUtil.java (hunk 1 above)
Comment (nandeeshp24) [reply backend-server/src/main/java/org/dice/ida/util/RDFUtil.java]: taken care of

**Review by nikit91 — APPROVED** [review]
lgtm

**Pull request merged**